frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -949,6 +955,7 @@ export const dashboardLogic = kea<dashboardLogicType>({
             await breakpoint(200)
             await api.update(`api/projects/${values.currentTeamId}/dashboards/${props.id}`, {
                 filters: values.filters,
+                no_items_field: true,
```

Comment:
This repetition of `no_items_field` points at an overdue move of this logic's API calls into `api.ts`

Only handle this here if you want to though... there are lots of overdue improvements in this file 😆